packages/create-plugin/src/additions/additions.ts

```diff
@@ -0,0 +1,10 @@
+import { Codemod } from '../codemods/types.js';
+import { resolveScriptPath } from '../utils/utils.codemod.js';
+
+export default [
+  {
+    name: 'example-addition',
+    description: 'Example addition demonstrating Valibot schema with type inference',
+    scriptPath: resolveScriptPath(import.meta.url, './scripts/example-addition.js'),
```

Comment:
scripts are now imported from the shared codemod runner, so paths must be resolved relative to this registry file's location to ensure correct resolution at runtime